src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/service/LimitedPartnershipServiceTest.java

```diff
@@ -75,8 +79,8 @@ void givenDto_whenCreateLP_thenLPCreatedWithSubmissionIdAndTransactionUpdated()
         // assert transaction resources are updated appropriately
         Transaction sentTransaction = transactionApiCaptor.getValue();
         assertEquals(limitedPartnershipSubmissionDto.getData().getPartnershipName(), sentTransaction.getCompanyName());
-        assertNull(sentTransaction.getCompanyNumber());
-        String submissionUri = String.format("/transactions/%s/limited-partnership/%s", transaction.getId(), limitedPartnershipSubmissionDao.getId());
+        assertNull(sentTransaction.getCompanyNumber());  
```

Comment:
Spaces at the end of the line here and on line 83.

Reply:
space removed